utils.php

```
@@ -1,53 +1,200 @@
-    curl_close($curl);
-    if($debug == 'yes') {
-		jsprint($response);
-	}
+/**
+ * Calls wp_remote_post or wp_remote_get with up to $max_retries additional
+ * attempts (1s backoff between) when the first attempt looks like a failure.
+ * Returns the last response.
+ */
+function xpay_http_request($url, $args, $method, $max_retries = 1) {
+    $callable = ($method === 'POST') ? 'wp_remote_post' : 'wp_remote_get';
+    $response = $callable($url, $args);
```

PHPStan (2.1.51)

[error] Trying to invoke 'wp_remote_get'|'wp_remote_post' but it's not a callable.


(callable.nonCallable)